source/usage-examples/findOne.rst

```diff
@@ -0,0 +1,44 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+You can find a single document using the ``collection.findOne()``
+method. If no document is found, ``findOne`` returns ``null``. If a
+document is found, ``findOne`` returns that document as an Object.
+The following snippet finds a single document from the ``movies``
+collection:
+
+.. code-block:: javascript
+
+   const { MongoClient } = require('mongodb');
+
+
+   const client = new MongoClient(
+      'mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority',
+   );
+
+   async function run() {
+      try {
+         await client.connect();
+         const database = client.db('sample_mflix');
+         const collection = database.collection('movies');
+         const movie = await collection.findOne({ title: 'The Room' });
+         // since this method returns the matched document, not a cursor, print it directly
```

Comment:
This seems like it needs more context. Naively, what is a cursor and why would I think that this might return one?
Maybe we could have a comment that says: "findOne() returns a Promise that resolves to either the matched document or null if no document matched the query"

Reply:
Going to leave this comment the same, but I'm adding a note about Cursors to the overview text.